### PR TITLE
Car validation fix

### PR DESCRIPTION
**Pull request by alexeyoverchenko**
small fixes for car

### CODE REVIEW

**Review by IrynaZazyba — APPROVED** [review]

File: frontend/src/index.css [hunk 1]
```diff
@@ -299,3 +299,7 @@ th.accept-all-item:hover {
 .page-size{
     float: right;
 }
+
+.page-size{
+    float: right;
+}
```
Comment (FortuneAwaker) [inline frontend/src/index.css]: dublicate

File: frontend/src/pages/car/Cars.js [hunk 2]
```diff
@@ -172,7 +172,10 @@ export default () => {
                 {tableRows}
                 </tbody>
             </Table>
-            <Page page={page} onChange={changePage}/>
+            {(cars.length > 0) &&
+            <Page page={page} onChange={changePage}/>}
+            {cars.length == 0 &&
+            <span>Empty list of items.</span>}
```
Comment (FortuneAwaker) [inline frontend/src/pages/car/Cars.js]: add the same thing before table

File: frontend/src/pages/warehouse/ModalEditWarehouse.js [hunk 3]
```diff
@@ -304,19 +304,19 @@ function ModalEditWarehouse(props) {
                     {errors.serverErrors && <ErrorMessage message={errors.serverErrors}/>}
                     <Form>
                         <Form.Group controlId="identifier" style={{padding: '5px 10px'}}>
-                            identifier
+                            Identifier
                             <Form.Control type="text"
                                           disabled
```
Comment (FortuneAwaker) [inline frontend/src/pages/warehouse/ModalEditWarehouse.js]: Add placeholder to every form, please.

File: frontend/src/pages/warehouse/Warehouses.js [hunk 4]
```diff
@@ -197,7 +203,10 @@ export default (props) => {
                 {tableRows}
                 </tbody>
             </Table>
-            <Page page={page} onChange={changePage}/>
+            {(warehouses.length > 0) &&
+            <Page page={page} onChange={changePage}/>}
+            {warehouses.length == 0 &&
+            <span>Empty list of items.</span>}
```
Comment (FortuneAwaker) [inline frontend/src/pages/warehouse/Warehouses.js]: Add the same thing for table

**Review by FortuneAwaker — COMMENTED** [review]
See comment, please.

**Review by FortuneAwaker — APPROVED** [review]

**Pull request merged**